src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -160,6 +163,19 @@ private void readFromNBT(NBTTagCompound compound)
             }
         }
 
+        //todo update fields from tileEntity.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)